spec/queries_spec.rb

```diff
@@ -51,7 +51,7 @@
         user.properties(:dashboard).bar = 'string'
         user.properties(:calendar).bar = 'string'
         user.save!
-      }.to perform_queries(3)
+      }.to perform_queries(4)
```

Comment:
The number of queries increases because we're not loading of the target's property objects into memory anymore. So we have one query to get the user's `:dashboard` property object and another to get the user's `:calendar` property object.